### PR TITLE
Don't send empty and blank messages

### LINKED ISSUE #62: Empty message gets sent.
Clicking the send button even if the message is empty sends the message. Please disable the send button untill there is something in the text field or just dont send the message whenever the click happens.

### PR DESCRIPTION
**Pull request by therohitramesh**
If the message box is empty or has only blank spaces the message doesn't get sent by clicking enter.

Fixes #62

### CODE REVIEW

**Comment by therohitramesh** [comment]
Deployment link: https://omeclone-therohitramesh.herokuapp.com/chat
@sumedh123 please check.

File: public/js/chat.js [hunk 1]
```diff
@@ -22,9 +22,10 @@
   });
 
   sendbtn.addEventListener('click', () => {
-    console.log(`Sending message to ${room_id_of_other_user}`);
-    socket.emit('sendMessage', { "room": room_id_of_other_user, "message": message.value });
-    message.value = ' ';
```
Comment (sumedh123) [inline public/js/chat.js]: why did you remove this?

File: public/js/chat.js [hunk 2]
```diff
@@ -22,9 +22,10 @@
...
+    if((message.value.trim()).length!==0){
```
Comment (sumedh123) [inline public/js/chat.js]: space between ```length``` and ```0```

Comment (sumedh123) [inline public/js/chat.js]: space between bracket and curly.

File: public/js/chat.js (hunk 1 above)
Comment (sumedh123) [reply public/js/chat.js]: insert that back.

Comment (therohitramesh) [reply public/js/chat.js]: I haven't removed it. I have just added it inside an if statement's true part.

**Comment by therohitramesh** [comment]
@sumedh123 Deployment link: https://omeclone-therohitramesh.herokuapp.com/chat

File: public/js/chat.js (hunk 1 above)
Comment (sumedh123) [reply public/js/chat.js]: the deployment dint replicate. thats why I asked.
